docs/conf/portland/2020/news/announcing-dates-tickets.rst

```diff
@@ -0,0 +1,45 @@
+:template: {{year}}/generic.html
+
+.. post:: May 14, 2020
+   :tags: portland-2020, speakers, tickets, visiting, sponsors
+
+Announcing ticket sales and dates
+=================================
+
+As we've :doc:`announced </blog/moving-portland-prague-2020-online>`, Write the Docs Portland 2020 will be a virtual conference.
+
+We've decided to keep the dates as previously announced: the writing day is on
+**{{date.day_two.dotw}}, {{date.month}} {{date.day_two.date}}, 9am-6pm {{tz}}**,
+and the main conference days on **{{date.day_three.dotw}}, {{date.month}} {{date.day_three.date}}, 9am-6pm {{tz}}**
+
+We have reopened :doc:`ticket sales </{{shortcode}}/{{year}}/tickets>`. Your virtual ticket includes:
```

Comment:
I feel like opening tickets on a Monday sounds better, and we also need to prepare more things around grants and whatever, so I figured it's better to have a few more days rather than too few ;)